packages/sanity/src/core/studio/components/navbar/search/components/common/FilterLabel.tsx

```diff
@@ -94,6 +94,10 @@ function getFilterValues(filter: SearchFilter): SearchFilterValues {
   if (isStringOrNumber(filter.value)) {
     values.value = filter.value
   }
+  if (typeof filter.value === 'boolean') {
+    // Cast boolean into a string value
+    values.value = filter.value.toString()
```

Comment:
@binoy14 I think it would be better for us to use the i18n string for true and false so that it matches the options in the dropdown if it is in a different language

```suggestion
    values.value = filter.value ? t('search.filter-boolean-true') : t('search.filter-boolean-false')
```

Reply:
💡good catch! will update thanks!